### PR TITLE
new info

### PR DESCRIPTION
**Pull request by andytnt**
removed `tag` table

### CODE REVIEW

**Comment by pielco11** [comment]
You can still push commits without creating new pull requests

File: MANIFEST.in [hunk 1]
```diff
@@ -1 +1 @@
-include README.md LICENSE
+include README.md LICENSE
```
Comment (pielco11) [inline MANIFEST.in]: what

File: LICENSE [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (pielco11) [inline LICENSE]: I don't understand why there are these changes

**Comment by pielco11** [comment]
Quickly looking at the code there are useless changes, which are confusing me

Please cleanup

**Comment by alexandreteles** [comment]
Okay, that's a commit really hard to read...

**Pull request closed without merging**